### DISCUSSION TITLE
Difference between GitHub release and Store version?

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [microsoft/WinUI-Gallery#1241] [Q&A] [answered]
Hey, when I look at the code page, I see _Latest (released) version 1.3.16, Nov 11 '22_ but via the Store I keep getting regular updates. How's that?

**Comment by marcelwgn** [comment]
The store updates are not reflected on the github releases page since they are not built with a CI that also reports these changes to the GitHub repo.

**Reply by Jay-o-Way** [reply]
So, does that mean the GitHub version and the Store version are out of sync?

**Reply by marcelwgn** [reply]
Disclaimer: I'm not an employee of Microsoft nor do I push updates for this app to the Microsoft Store.

Every commit on the main branch will eventually land in the store version assuming they are not reverted, however a PR merged today might not be present in the store version for the next few weeks or even months depending when an update is released to the store.

**Reply by Jay-o-Way** [reply]
Hm, looking at the data I would've guessed the opposite: GitHub shows a _latest version_ of 5 months ago... Maybe @niels9001 knows something more?

**Reply by marcelwgn** [reply]
Well depends on what you mean by "GitHub version" here. If you mean the code on the main branch, then they are not out of sync. If you mean the github releases, yes they are out of sync.

**Reply by karkarl** [reply, marked as answer]
Hey @Jay-o-Way! You are absolutely correct that it's out of sync. This will be a prioritized work item for this quarter. I recently updated the project board with the targeted work items, take a look if you're interested / have any feedback. #1025 via [projects](https://github.com/orgs/microsoft/projects/368/views/1)

**Comment by Jay-o-Way** [comment]
Time for a new release! :)
